library/Ivoz/Api/Swagger/Metadata/Resource/ResourceMetadata/FilterMetadataFactory.php

```diff
@@ -74,16 +74,18 @@ private function getEntityFilters(string $resourceClass)
 
         $attributes = $this->getEntityAttributes($resourceClass);
         foreach ($attributes as $attribute) {
+            $type = $this->getFieldType($resourceClass, $attribute);
+            if (!is_null($type)) {
+                $filters['ivoz.api.filter.order'][$attribute] = Filter\OrderFilter::NULLS_LARGEST;
```

Comment:
Have you considered creating constants for 'ivoz.api....' strings?

Reply:
Done :)